docs/versioning/how-to-view-versions.md

```diff
@@ -0,0 +1,107 @@
+
+|                                    Explore Lineage in Provenance Store                                     |
+| :--------------------------------------------------------------------------------------------------------: |
+| <img src="../assets/images/versioning/versioning-registry-explore-lineage.png" alt="drawing" width="600"/> |
+
+To view previous versions, you can use your mouse to double-click on the item node to expand other related nodes.
+
+If you continue expanding the nodes including the version activity node, and respective items, you will be able to trace its version lineage in the provenance graph viewer as shown below.
+
+|                             Explore version provenance lineage graph                              |
+| :-----------------------------------------------------------------------------------------------: |
+| <img src="../assets/images/versioning/prov-store-version-lineage.gif" alt="drawing" width="600"/> |
+
+## Viewing version info in the Data Store
+
+### Step 1. Navigate to Versioning tab in the dataset record in the Data Store
+
+-   Ensure you're logged in.
+-   Navigate to the data store.
+-   Navigate to the dataset record.
+-   Click on the "VERSIONING" tab as shown below.
+
+|                                    Versioning Tab                                     |
+| :-----------------------------------------------------------------------------------: |
+| <img src="../assets/images/versioning/versioning-tab.png" alt="drawing" width="600"/> |
+
+You will be able to view the version history including the current and previous versions.
+
+|                                   Updated versioning details                                    |
+| :---------------------------------------------------------------------------------------------: |
+| <img src="../assets/images/versioning/versioning-create-result.png" alt="drawing" width="600"/> |
+
+You can click on the icon with 3 dots in the versioning timeline to expand to display additional versions.
+
+<img src="../assets/images/versioning/versioning-view-expand.png" alt="drawing" width="400"/>
+
+Sorting is available in the drop down menu.
+
+<img src="../assets/images/versioning/versioning-view-sort.png" alt="drawing" width="400"/>
+
+Clicking on the link of the ID of the version will enable you to navigate to that particular version.
+
+### Step 2. Viewing lineage
+
+If you would like to view the lineage graph of the dataset, click on the "LINEAGE GRAPH" tab to view the dataset lineage.
+
+You will see the direct lineage of the dataset in the provenance graph viewer as shown below.
+
+|                             Dataset lineage graph including direct versioning info                             |
+| :------------------------------------------------------------------------------------------------------------: |
+| <img src="../assets/images/versioning/versioning-registry-view-lineage-direct.png" alt="drawing" width="600"/> |
```

Comment:
This has the wrong file name and is not rendering on the deployed page. I think the correct file is "[../assets/images/versioning/versioning-registry-view-lineage.png](https://github.com/provena/provena/pull/16/files#diff-60b40641e98e28c4fb4e0f21ae07f91fea573d3d1704bd67e01693973190cb3e)"